lib/galaxy/selenium/navigation.yml

```diff
@@ -563,10 +563,16 @@ libraries:
       download_button: '#download-btn'
       delete_btn: '.toolbtn-bulk-delete'
       toast_msg: '.toast-message'
+      select_import_dir_item: 'li[full_path="${name}"] .jstree-anchor'
```

Comment:
Oof, I forgot jstree was here too and not just the toolshed -- one more thing to replace!

(this looks fine though)

Reply:
yeah, it's obsolete code from old library. user_import_dir utilizes the same